astroid/rebuilder.py

```diff
@@ -278,10 +278,9 @@ def visit_module(
 
         Note: Method not called by 'visit'
         """
-        node, doc_ast_node, doc = self._get_doc(node)
+        node, doc_ast_node, _ = self._get_doc(node)
```

Comment:
Anything against updating ` self._get_doc`, too? It's an internal method, so no need to keep backwards compatibility there.